src/stories/containers/ActorsAbout/ActorSummary/ActorTitleAbout.tsx

```diff
@@ -41,7 +41,7 @@ export const ActorTitleAbout = ({ actorAbout, showTextDescription }: Props) => {
                 <ResponsiveTitle>
                   {actorAbout?.name && <TypographyTitle isLight={isLight}>{actorAbout?.name}</TypographyTitle>}
                   <TypographyCategory isLight={isLight}>
-                    {pascalCaseToNormalString(actorAbout?.category[0] || '')}
+                    {pascalCaseToNormalString(actorAbout.category?.[0] ?? ('' || ''))}
```

Comment:
this `('' || '')` doesn't make too much sense, `?? ''` should be enough

Reply:
you are right